lambdas/indexer/app.py

```diff
@@ -458,3 +459,23 @@ def nudge(event: chalice.app.CloudWatchEvent):
         log.info('Nudging queue with %i token(s) for a total value of %i', len(tokens), num_documents)
         for batch in chunked(tokens, 10):
             _send_tokens(batch)
+
+
+@app.schedule('rate(5 minutes)')
+def retrieve_fail_messages(event: chalice.app.CloudWatchEvent):
+    """
+    Get all the messages from the fail queue and save them in the the DynamoDB failure message table.
+    """
+    assert event is not None  # avoid linter warning
+    messages = Queues().receive_all_messages(config.fail_queue_name)
```

Comment:
I would suggest something like the following. But is up to you.  
```suggestion
    queues = Queues()
    messages = queues.receive_all_messages(config.fail_queue_name)
```